fixtures/custom-src-paths/sku.config.ts

```diff
@@ -1,9 +1,12 @@
 import type { SkuConfig } from 'sku';
+// @ts-ignore no types
```

Comment:
"Why not `ts-expect-error`?" you might ask. Because typescript-eslint and ts-jest disagree on the severity. We have [`strict: false`][false] in our monorepo and [`strict: true`][true] in fixtures.

[false]: https://github.com/seek-oss/sku/blob/ddd9ac1d571a0b02d1c8719cd7a655befd32430b/tsconfig.json#L7
[true]: https://github.com/seek-oss/sku/blob/9620c75b3d36bbcc48d6ce0634c8e9ca395a7b3a/packages/sku/config/typescript/tsconfig.js#L29